crates/wysiwyg/src/composer_model/lists.rs

```diff
@@ -274,11 +277,25 @@ where
             if let DomNode::Text(t) = node {
                 let text = t.data();
                 let index_in_parent = handle.index_in_parent();
+                let add_zwsp = !text.to_string().starts_with("\u{200b}");
```

Comment:
Since you have `TextNode.add_leading_zwsp`, couldn't that be used instead?

Reply:
Yeah I plan to use that when it is structured a bit better, right now the TextNode is re-created instead of moved and we have a non-mut context for it. I'll check if it is an easy fix, otherwise keeping it for later 😉